be/src/util/jsonb_stream.h

```diff
@@ -126,9 +130,40 @@ class JsonbOutStream : public std::ostream {
         size_ += len;
     }
 
-    void write(__int128 l) {
-        // TODO
-        assert(false);
+    // TODO: use more efficient impl for int128
+    void write(int128_t l) {
```

Comment:
maybe we could use
```
fmt::memory_buffer buffer;
fmt::format_to(buffer, "{}", l);
```

Reply:
changed to `fmt::format_to_n(head_ + size_, MAX_INT128_DIGITS, "{}", l)`